packages/file-adapters/README.md

```diff
@@ -48,3 +48,37 @@ const fileAdapter = new CloudinaryAdapter({
 | `apiKey`    | `String` | Required    |             |
 | `apiSecret` | `String` | Required    |             |
 | `folder`    | `String` | `undefined` |             |
+
+## `S3FileAdapter`
+
+```javascript
+const { S3Adapter } = require('@keystonejs/file-adapters');
+
+const fileAdapter = new S3Adapter({
+  accessKeyId: 'ACCESS_KEY_ID',
+  secretAccessKey: 'SECRET_ACCESS_KEY',
+  region: 'us-west-2',
+  bucket: 'bucket-name',
+  folder: '/cat-photos',
+  publicUrl: ({ id, filename, _meta }) => `https://{CF_DISTRIBUTION_ID}.cloudfront.net/cat-photos/${filename}`,
```

Comment:
Should this be `${CF_DISTRIBUTION_ID}`?

Reply:
It was meant as a placeholder but I updated it to be a constant.